lib/rules/valid-test-filenames.js

```diff
@@ -24,18 +25,26 @@ module.exports = {
     let reported = false;
 
     const error = (context) => (node) => {
-      const { testSuffix, whitelist } = context.options[0];
+      const { testSuffixes, whitelist } = context.options[0];
       const filename = context.getFilename();
 
       if (reported) { return; }
       if (whitelist.some((allowedFilename) => filename.endsWith(allowedFilename))) { return; }
-      if (filename.endsWith(testSuffix)) { return; }
+      if (testSuffixes.some((suffix) => filename.endsWith(suffix))) { return; }
 
       reported = true;
-      context.report({
-        node,
-        message: `Test filenames must end with ${testSuffix}`,
-      });
+
+      if (testSuffixes.length === 1) {
+        context.report({
+          node,
+          message: `Test filenames must end with ${testSuffixes[0]}`,
```

Comment:
Isn't this case unnecessary, since if the length is 1 then `testSuffixes.join(' or ')` will be equal to `testSuffixes[0]`?

```
>> ['a', 'b'].join(' or ')
<- "a or b"

>> ['a'].join(' or ')
<- "a"
```

Reply:
Not sure why I didn't think of this I'll go ahead and make this change 👍